client/app.jsx

```diff
@@ -0,0 +1,31 @@
+// 1. Component to render sample data
+// 2. Refactor to use data from route calls
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      bundles: []
+    }
+  }
+
+  componentDidMount() {
+    // this.setState to update bundles from external server
+  }
```

Comment:
This is skeleton code, so not a big deal, but please comment out any unused methods/ functions.